dae-alright/src/components/Navigation/Navigation.jsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
```

Comment:
You can rename this file to `index.jsx` since its name matches its parent folder name